app/Resources/views/layout.html.twig

```diff
@@ -166,7 +166,7 @@
 <script type="text/javascript" language="javascript">
 $(function() {
 
-  initSearchBar('{{ path('uvweb_uv_nameslike') }}', '/unis/', '{{ path('uvweb_uv_detail') }}');
+  initSearchBar('{{ path('uvweb_uv_nameslike') }}', '{{ path('uvweb_foreignclass_details', { 'uvname': 'uvname', 'uniId': 'uniId' }) }}', '/unis/', '{{ path('uvweb_uv_detail') }}');
```

Comment:
My bad, left the parameter in excess after a branch separation. I'll update now.